Makefile

```diff
@@ -6,53 +6,46 @@ PORT ?= 3000
 
 .PHONY: dev
 dev:
-	HKNWEB_MODE='dev' pipenv run python ./manage.py runserver $(IP):$(PORT)
+	HKNWEB_MODE='dev' python ./manage.py runserver $(IP):$(PORT)
```

Comment:
I'd suggest you still prefix this with the virtualenv location so that you don't get people accidentally using their system python if they haven't activated the virtualenv. Essentially like what was in f7ef64963aab581e53e8beb1b99c46d72c8b899d before #4, but don't worry about using venv-update.

Reply:
Yeah, something like `<venv>/bin/python`.

Reply:
bless f7ef649, added prefixes